client/gutenberg/extensions/subscriptions/index.js

```diff
@@ -0,0 +1,43 @@
+/** @format */
```

Comment:
If you rebase with latest `master`, these pragmas won't be needed anymore :tada:

(ref https://github.com/Automattic/wp-calypso/pull/29403)